group-aws/README.md

```diff
@@ -9,12 +9,25 @@ There are currently no regular meetings held. The IRC channel is the
 main and official place to contact the members. For specific issues and
```

Comment:
Well, this README.md file should be concise and give pointers to other locations.
In the Openstack community this file is actually auto-generated, and we will probably do the same thing in the future.
So I am not sure the Working Group Goals belong in this file.

(The OpenStack community does allow custom content in this file, but that's at the very end of the file IIRC)